### PR TITLE
Fix for rails 6

### PR DESCRIPTION
**Pull request by jvalol**

### CODE REVIEW

**Comment by diclophis** [comment]
@brentsondgeroth / @guanw88 yea, this needs to get merged as well... gonna tack it on the bigmaven side with a ref update

**Review by brentsondgeroth — APPROVED** [review]

**Comment by arthurlm44** [comment]
In bigmaven, secret_token returns `nil`.
```RUBY
irb(main):001:0> Rails.application.config.secret_token
=> nil
irb(main):002:0> Rails.application.config.secret_key_base
=> "0ac1f9b56d60dd6cc418bab[REDACTED SHASUM]efa1ade33da54"
```

I think either way, this work. We checked on the QBWC upstream gem, and they `.try` for both secret_token, and secret_key_base. So whichever we want to use is probably fine.


https://github.com/qbwc/qbwc/blob/master/lib/qbwc/session.rb#L21

**Review by arthurlm44 — APPROVED** [review]
LGTM

**Review by guanw88 — APPROVED** [review]
LGTM. Do we use `config.secret_token` anywhere else?

File: lib/qbwc/session.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def initialize(user = nil, company = nil, ticket = nil)
     @iterator_id = nil
     @initial_job_count = pending_jobs.length
 
-    @ticket = ticket || Digest::SHA1.hexdigest("#{Rails.application.config.secret_token}#{SecureRandom.uuid}#{Time.now.to_f}")
```
Comment (ethransom) [inline lib/qbwc/session.rb:23]: Context here is that `secret_token` will begin to throw in rails6

File: lib/qbwc/session.rb [hunk 2]
```diff
@@ -20,7 +20,7 @@ def initialize(user = nil, company = nil, ticket = nil)
...
+    @ticket = ticket || Digest::SHA1.hexdigest("#{Rails.application.config.secret_key_base}#{SecureRandom.uuid}#{Time.now.to_f}")
```
Comment (brentsondgeroth) [inline lib/qbwc/session.rb]:
```suggestion
    @ticket = ticket || Digest::SHA1.hexdigest("#{SecureRandom.uuid}")
```

**Pull request merged**